shell/renderer/api/electron_api_ipc_renderer.cc

```diff
@@ -83,9 +87,14 @@ class IPCRenderer : public gin::Wrappable<IPCRenderer>,
 
  private:
   void SendMessage(v8::Isolate* isolate,
+                   gin_helper::ErrorThrower thrower,
```

Comment:
If we're introducing an `ErrorThrower` we can pull the isolate off of it with `thrower.isolate()` - we probably don't need to keep the isolate as a separate param

Reply:
True, though it doesn't seem particularly compelling to me to do so. Do you feel strongly about it?

Reply:
not per se but we've preferred to consolidate and avoid excess isolates everywhere else in the codebase so this is a bit of a break from existing style